### DISCUSSION TITLE
Bug: Extra bottom padding in React Mux Player

### DISCUSSION THREAD

**Discussion by ywong-cais** [muxinc/elements#452] [Q&A] [answered]
### Checklist

- [x] Verified whether your issue exists in the latest version Mux Element(s) release: `npm install @mux/[element_name]` or `yarn @mux/[element_name]`.
  - (NOTE: If the issue is not occurring in the latest version, we may not resolve it.)
- [x] Confirmed that this issue hasn't [already been filed](https://github.com/muxinc/elements/issues?q=is%3Aopen+is%3Aissue+label%3Abug).

### Which Mux Elements/Packages has a bug?
mux-player-react

### Which browser(s) are you using?
Chrome Version 105.0.5195.125

### Which operating system(s) are you using?

(For example: macOS, Windows, iOS, Android)

### How are you using Mux Elements?
yarn install

### Describe the bug
Trying to give the player a border radius by using a wrapper div with following styles
```
{borderRadius: 30, overflow: 'hidden'}
```
but there is a white padding below the player. 
<img width="623" alt="image" src="https://user-images.githubusercontent.com/111991837/194519444-f8c0daf8-4eb9-40cf-9a5a-7cd41b694884.png">

## Steps To Reproduce
- Create a new react project
- yarn add @mux/mux-player-react
- Render a player using the following code
```
      <div style={{borderRadius: 30, overflow: 'hidden'}}>
        <MuxPlayerReact
          playbackId="DS00Spx1CV902MCtPj5WknGlR102V5HFkDe"
          metadata={{
            video_id: 'video-id-123456',
            video_title: 'Bick Buck Bunny',
            viewer_user_id: 'user-id-bc-789',
          }}
          streamType="on-demand"
        />
      </div>
```
- notice bottom padding

Link to code example:

## The current behavior
player has bottom padding

## The expected behavior
should have no bottom padding

**Comment by luwes** [comment, marked as answer]
Thanks for the feedback @ywong-cais! 

This might feel a bit counter intuitive but the `<mux-player>` element has an intentional `display` CSS property as `inline-block`.
It's to keep it more aligned with how the native `<video>` element behaves.

The good news is that if you don't like the video player to behave as an `inline` element, simply set the `display` property to `block`.
e.g. `<MuxPlayerReact style={{ display: 'block' }} />`

That should remove the extra padding at the bottom.

--- 

Also it's recommended that you define explicit dimensions to your player to avoid cumulative layout shift.
See the docs for more info https://docs.mux.com/guides/video/mux-player-customize-look-and-feel#aspect-ratio

Hope this helps!
